source/chapters/microphones.rst

```diff
@@ -78,26 +83,32 @@ To setup direct monitoring, it must be
 :ref:`activated on your soundcard <microphones-activating-direct-monitoring-on-soundcard>`.
 If you are not recording or broadcasting the microphone with the music, no
 further setup is needed. To record and/or broadcast with the microphone
-signal mixed into the music, setting up a either a
-:ref:`loopback input <microphones-loopback-input>` or
-:ref:`latency compensation <microphones-latency-compensation>` is required.
+signal mixed into the music, set up a either a 
+:ref:`loopback input<microphones-loopback-input>` or
+:ref:`latency compensation <microphones-latency-compensation>`.
+
+Because the microphones are mixed with the music by the soundcard rather 
+than Mixxx, adjust the microphone volume with the input gain knob on 
+your soundcard. If you adjust the microphone gain knob in Mixxx, the volume 
+of the microphone relative to the music will be different in your recorded 
+and broadcasted mixes compared to what you hear from your soundcard's output.
 
 .. _microphones-loopback-input:
 
 Loopback Input
 ^^^^^^^^^^^^^^
 Some soundcards that support direct monitoring have a feature called
-"loopback". Using a soundcard with this feature is the recommended way
-to use microphones with Mixxx because it is the easiest and cheapest to setup
-without the latency issue of
-:ref:`software monitoring <microphones-software-monitoring>` and with good
-sound quality.
-
-Using direct monitoring, the soundcard mixes the microphone input with the
-music output from Mixxx in hardware. This avoids the latency of sending the
+"loopback". Using direct monitoring with a soundcard that supports loopback is 
+the recommended way to use microphones with Mixxx. It is the easiest to 
+setup without the latency issue of
+:ref:`software monitoring <microphones-software-monitoring>` and cheapest 
+to set up with good sound quality.
+
```

Comment:
Rephrase: It is the easiest and cheapest to setup, without the latency issue of software monitoring and with good sound quality.